### ISSUE TITLE
Unreal Editor Log tab in rider don't show when project build finished.

### ISSUE THREAD

**Issue by nameless-user12138** [JetBrains/UnrealLink#263]
Version: Rider 2023.1.3, Unreal engine 5.1.1, RiderLink Plugin 2023.1.0.282

What happend:
1. Unreal link tab and it's sub tab Unreal Editor Log disappeared when project build finished,
2. Pop up window "RiderLink plugin is required" is not Pop up when rider is start,
3. In **Settings | Languages & Frameworks | Unreal Engine,** all the bottom "Install RiderLink In Engine", "Install RiderLink In Game", "Extra RiderLink In Engine", "Extra RiderLink In Game", is not clickable(it's not reaction), that means I event can't force build and install rider link.

Weired things:
1. Only one of my project has the issue above, other projects Unreal link tab and it't sub tag Unreal Editor Log works fine, this project once have plugin/developer/riderlink, I have delete all the file in this folder , and check all the reference in project, the problem going on.
2. It's seems that unreallink's other utilities still work, such as count the blueprint subclass of the c++ parent class, count usages about ufunction with blueprintcallalbe, that utilities are provided by rider's native plugin riderlink?
 
![IMG_20230613_163128](https://github.com/JetBrains/UnrealLink/assets/54355449/796dc356-4f23-4b2d-8a29-e8211f62900d)

**Comment by osolovyoff** [comment]
Yeah! I have the same issue, It looks like they redirect output to some log file into JetBrains folder
